src/tests/Register.test.js

```diff
@@ -64,4 +65,80 @@ it('allows the user to register successfully', async () => {
 
     expect(screen.getByLabelText('response')).toHaveTextContent("User was created successfully. A confirmation email has been sent via email. After confirming your email you can login.")
     
-})
\ No newline at end of file
+})
+
+it('Check empty field warning', async () => {
+        
+    const { getByLabelText }= render(<Register />)
```

Comment:
You can either declare as cons at the beginning to be used in the test or don't declare at all and use`screen.getByLabelText...` all the way. But to be consistent with the first test (it allows user to register succcessfully) keep it as `screen.getByLabelText`.  Keeping consistency in how the code is written across the project helps improve code readability.